.github/workflows/hardware-long.yml

```diff
@@ -18,10 +18,13 @@ jobs:
         config:
           - board: p100
             runs-on:
-              - yyz-zephyr-lab-p100
```

Comment:
On second glance, and given that our runners are available to everyone in the org, do we maybe want to change the labels to be more specific so that it's clear they are used for zephyr firmware?

Eventually, we should be able to go fully generic anywhere on the cloud, but I think that it would require the "unbrickable" fw boot flow for CMFW.

Maybe until then we should just keep some kind of prefix.